README.md

```diff
@@ -1,8 +1,46 @@
 # AWS Java Toolkit
 
-## Java for abstracting basic actions in AWS
+## A collection of small Java libraries for abstracting common operations on Amazon Web Services (AWS).
+
+### Requirements: 
+ 1. Java 17 or newer
+ 2. AWS Java SDK version 2
+ 3. Jackson (https://github.com/FasterXML/jackson)
+
+## Usage:
+The library is available in Maven Central. 
+Include it as a dependency in your project:  
+Example:
+```
+    <dependency>
+    <groupId>io.github.awsjavakit</groupId>
+    <artifactId>misc</artifactId>
+    <version>0.13.2</version>
+    </dependency>
+```   
+**Important Note:**   
+We are currently supporting only the specified versions of all the libraries that 
+aws-java-kit depends on. Different versions can produce a build error on the client's end.
```

Comment:
`Different versions can produce a build error on the client's end.` -> `Different versions in client code may produce build errors`